lib/stripe_mock/request_handlers/subscriptions.rb

```diff
@@ -137,8 +137,13 @@ def update_subscription(route, method_url, params, headers)
         end
 
         # expand the plan for addition to the customer object
-        plan_name = params[:plan] if params[:plan] && params[:plan] != {}
-        plan_name ||= subscription[:plan][:id]
+        plan_name = 
```

Comment:
and what if compress it to `params[:plan].is_a?(String) ? params[:plan] : subscription[:plan][:id]` ?